smoke-test/tests/telemetry/telemetry_test.py

```diff
@@ -5,7 +5,7 @@
 
 def test_no_clientID():
     client_id_urn = "urn:li:telemetry:clientId"
-    aspect = ["telemetryClientId"]
+    aspect = ["clientId"]  # this is checking for the removal of the invalid aspect RemoveClientIdAspectStep.java
```

Comment:
@RyanHolstien - Please validate this logic

Reply:
This is sort of valid, the test originally checked whether telemetry was being enabled during smoke tests (it shouldn't be). Really neither of these should show up, so ideally this would be an _additional_ test and not a replacement.